### PR TITLE
Data Tracker sync continued

### PR DESCRIPTION
**Pull request by mddilley**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/15550

This adds a helper script to evaluate the sync between Moped projects and Data Tracker `projects` table and delete Knack records that have no connection to a Moped project and no connection to other Knack records.

Also, this is a tool for the toolbox. It is not going to be part of the ETL.

## Testing
**URL to test:** 

n/a

**Steps to test:**
Hard and not really useful to test this on staging so I'm going to say the best way is to check out the code. It is unlikely that we will have to run this ever again.

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Review by chiaberry — APPROVED** [review]
reviewed the code, thanks for adding the code to the toolbox!

File: moped-etl/data-tracker-sync/data_tracker_sync.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ def main(args):
 
         # Update Moped project with Knack record ID of created record
         logger.info(
-            f"Updating Knack ID {knack_record_id} from Moped project {moped_project_id}"
+            f"Updating Moped project {moped_project_id} with Knack ID {knack_record_id}"
```
Comment (mddilley) [inline moped-etl/data-tracker-sync/data_tracker_sync.py:197]: This comment had the logic backwards. We are updating the Moped record with the ID from the newly created Knack record.

**Comment by mddilley** [comment]
@chiaberry thanks for the review! i just added some better logging in the last commit that I think make the script logging easier to follow. I found myself having to look at the script code to see what step the ETL was at when running manually from the Airflow side. Hopefully, this makes that better. 🤞

**Review by johnclary — APPROVED** [review]
looks great to me @mddilley! thank you! 🚢

**Comment by mddilley** [comment]
thanks y'all, merging so we can get the improved logging running in the ETL. 🙏 🚀

**Pull request merged**